### PR TITLE
Support for live locations

### PR DESCRIPTION
**Pull request by DTRandom**
Added support for live locations

### CODE REVIEW

File: docs/api/telegram.rst [hunk 1]
```diff
@@ -418,8 +418,12 @@ about its business.
       The *notify* parameter is for defining if your message should trigger
       a notification on the client side (yes by default).
 
+      The *live_period* parameter is for defining if this location must be a live location and needs to be updated over time.
+      Leave to None if it is not or set it as a number between 60 and 86400 (seconds) if it is.
```
Comment (MarcoBuster) [inline docs/api/telegram.rst]: Add ` around None (here and in the next occurrences)

File: docs/api/telegram.rst [hunk 2]
```diff
@@ -1895,6 +1911,35 @@ about its business.
...
+      :param object attach: An extra thing to attach to the message.
+      :param object extra: An extra reply interface object to attach.
+
+      .. deprecated:: 0.4
+
+         The *extra* parameter is now deprecated
+
+      .. versionadded:: 0.7
+
+
```
Comment (MarcoBuster) [inline docs/api/telegram.rst]: Only one newline between two methods

File: docs/changelog/0.7.rst [hunk 3]
```diff
@@ -19,11 +19,12 @@ Release description not yet written.
 New features
 ------------
 
-* Enhancements in message forwarding
+* Added support for live locations
```
Comment (MarcoBuster) [inline docs/changelog/0.7.rst]: Please add your changelog _after_ the one is in the master branch

File: botogram/objects/mixins.py [hunk 4]
```diff
@@ -274,13 +274,18 @@ def send_file(self, path=None, file_id=None, url=None, reply_to=None,
                               expect=_objects().Message)
 
     @_require_api
-    def send_location(self, latitude, longitude, reply_to=None, extra=None,
+    def send_location(self, latitude, longitude, live_period=None, reply_to=None, extra=None,
                       attach=None, notify=True):
-        """Send a geographic location"""
+        """Send a geographic location, set live_period to a number between 60 and 86400 if it's a Live Location"""
```
Comment (MarcoBuster) [inline botogram/objects/mixins.py]: Please be consistent: _live location_ should be lowercase

File: docs/changelog/0.7.rst (hunk 3 above)
Comment (DTRandom) [reply docs/changelog/0.7.rst]: Sorry, ctrl+v in the wrong spot!

**Comment by MarcoBuster** [comment]
@DTRandom Please gpg-sign all your commits, 52b08ae is not signed.

**Pull request merged**